client/components/app-promo/style.scss

```diff
@@ -1,44 +1,44 @@
 @import "@wordpress/base-styles/breakpoints";
 @import "@wordpress/base-styles/mixins";
 
-.go-mobile {
+.app-promo {
 	&.customer-home__card.card {
 		border-bottom: none;
 
 		@include break-large {
-			padding-left: 48px;
```

Comment:
I think the card still needs this (or at least some) left padding. This is emphasized by how poorly it looks without left padding in the devdocs example. 

Then, when embedded in the customer home, we could reset the left padding to `0`.

Reply:
Good suggestion. I update the code to revert the padding.